Reference app get crashed on Add Patient submit action.

**Describe the bug**
1. Launch reference app.
2. Click on floating + action button. -> It launches Add Patient screen.
3. Fill all data in Add Patient form.
4. Click on Submit.
5. Application is crashed with below exception.

java.lang.reflect.InvocationTargetException
        at java.lang.reflect.Method.invoke(Native Method)
        at com.google.android.fhir.datacapture.mapping.ResourceMapperKt.updateFieldWithEnum(ResourceMapper.kt:340)
        at com.google.android.fhir.datacapture.mapping.ResourceMapperKt.access$updateFieldWithEnum(ResourceMapper.kt:1)
        at com.google.android.fhir.datacapture.mapping.ResourceMapper.extractField(ResourceMapper.kt:298)
        at com.google.android.fhir.datacapture.mapping.ResourceMapper.extractFields(ResourceMapper.kt:220)
        at com.google.android.fhir.datacapture.mapping.ResourceMapper.extractField(ResourceMapper.kt:277)
        at com.google.android.fhir.datacapture.mapping.ResourceMapper.extractFields(ResourceMapper.kt:220)
        at com.google.android.fhir.datacapture.mapping.ResourceMapper.extractField(ResourceMapper.kt:254)
        at com.google.android.fhir.datacapture.mapping.ResourceMapper.extractFields(ResourceMapper.kt:220)
        at com.google.android.fhir.datacapture.mapping.ResourceMapper.extractByDefinitions(ResourceMapper.kt:128)
        at com.google.android.fhir.datacapture.mapping.ResourceMapper.extract(ResourceMapper.kt:104)
        at com.google.android.fhir.datacapture.mapping.ResourceMapper.extract$default(ResourceMapper.kt:101)
        at com.google.android.fhir.reference.AddPatientViewModel$savePatient$1.invokeSuspend(AddPatientViewModel.kt:53)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.internal.DispatchedContinuationKt.resumeCancellableWith(DispatchedContinuation.kt:377)
        at kotlinx.coroutines.intrinsics.CancellableKt.startCoroutineCancellable(Cancellable.kt:30)
        at kotlinx.coroutines.intrinsics.CancellableKt.startCoroutineCancellable$default(Cancellable.kt:25)
        at kotlinx.coroutines.CoroutineStart.invoke(CoroutineStart.kt:110)
        at kotlinx.coroutines.AbstractCoroutine.start(AbstractCoroutine.kt:126)
        at kotlinx.coroutines.BuildersKt__Builders_commonKt.launch(Builders.common.kt:56)
        at kotlinx.coroutines.BuildersKt.launch(Unknown Source:1)
        at kotlinx.coroutines.BuildersKt__Builders_commonKt.launch$default(Builders.common.kt:47)
        at kotlinx.coroutines.BuildersKt.launch$default(Unknown Source:1)
        at com.google.android.fhir.reference.AddPatientViewModel.savePatient(AddPatientViewModel.kt:52)
        at com.google.android.fhir.reference.AddPatientFragment.savePatient(AddPatientFragment.kt:98)
        at com.google.android.fhir.reference.AddPatientFragment.onSubmitAction(AddPatientFragment.kt:94)
        at com.google.android.fhir.reference.AddPatientFragment.onOptionsItemSelected(AddPatientFragment.kt:59)
        at androidx.fragment.app.Fragment.performOptionsItemSelected(Fragment.java:3111)
        at androidx.fragment.app.FragmentManager.dispatchOptionsItemSelected(FragmentManager.java:3222)
        at androidx.fragment.app.Fragment.performOptionsItemSelected(Fragment.java:3115)
        at androidx.fragment.app.FragmentManager.dispatchOptionsItemSelected(FragmentManager.java:3222)
        at androidx.fragment.app.FragmentController.dispatchOptionsItemSelected(FragmentController.java:416)
        at androidx.fragment.app.FragmentActivity.onMenuItemSelected(FragmentActivity.java:358)
        at androidx.appcompat.app.AppCompatActivity.onMenuItemSelected(AppCompatActivity.java:228)
        at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109)
        at androidx.appcompat.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:109)
        at androidx.appcompat.app.ToolbarActionBar$2.onMenuItemClick(ToolbarActionBar.java:65)
        at androidx.appcompat.widget.Toolbar$1.onMenuItemClick(Toolbar.java:207)
        at androidx.appcompat.widget.ActionMenuView$MenuBuilderCallback.onMenuItemSelected(ActionMenuView.java:779)
        at androidx.appcompat.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:834)
        at androidx.appcompat.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:158)
        at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:985)
2021-09-23 13:38:16.101 11432-11432/com.google.android.fhir.reference E/AndroidRuntime:     at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:975)
        at androidx.appcompat.widget.ActionMenuView.invokeItem(ActionMenuView.java:623)
        at androidx.appcompat.view.menu.ActionMenuItemView.onClick(ActionMenuItemView.java:151)
        at android.view.View.performClick(View.java:7448)
        at android.view.View.performClickInternal(View.java:7425)
        at android.view.View.access$3600(View.java:810)
        at android.view.View$PerformClick.run(View.java:28305)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:223)
        at android.app.ActivityThread.main(ActivityThread.java:7656)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
     Caused by: org.hl7.fhir.exceptions.FHIRException: Unknown ContactPointSystem code '1234567890'
        at org.hl7.fhir.r4.model.ContactPoint$ContactPointSystem.fromCode(ContactPoint.java:106)
        	... 56 more


**Component**
SDC library, reference app,


**Expected behavior**
App should not be crashed.

**Screenshots**


**Smartphone (please complete the following information):**
 - Device: Pixel XL emulator
 - Android version:  11

**Additional context**


**Would you like to work on the issue?**
Please state if this issue should be assigned to you or who you think could help to solve this issue.

--- a/reference/src/main/assets/new-patient-registration-paginated.json
+++ b/reference/src/main/assets/new-patient-registration-paginated.json
@@ -261,6 +261,10 @@
                     "expression": "ContactPoint$ContactPointSystem",
                     "name": "contactPointSystem"
                   }
+                },
+                {
+                  "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-hidden",
+                  "valueBoolean": true
                 }
               ],
               "type": "string",
@@ -268,13 +272,6 @@
               "initial": [
                 {
                   "valueString": "phone"
-                }
-              ],
-              "enableWhen": [
-                {
-                  "question": "patient-0-gender",
-                  "operator": "=",
-                  "answerString": "ok"
                 }
               ]
             },

